ddtrace/internal/runtime/metric_collectors.py

```diff
@@ -82,7 +82,7 @@ def collect_fn(self, keys):
                 metrics[metric] = delta
 
             # Populate metrics that just take instantaneous reading
-            for metric, fun in self.abs_funs.items():
+            for metric, func in self.abs_funs.items():
```

Comment:
```suggestion
            for metric, abs_func in self.abs_funs.items():
```
might be safer for the future? (and perhaps use `delta_func` above for good measure?)

Reply:
Very agreeable, made a corresponding change in 9b8163781eaff0496c9e35144164aa417d5515db